site/src/_pages/docs/0_index.mdx

```diff
@@ -1,12 +1,26 @@
 ## Overview
 
-These pages provide a practical introduction to implementing the block protocol, whether you are:
+The Block Protocol is a global standard for building interactive blocks connected to structured data.
 
-- interested in [developing blocks](https://blockprotocol.org/docs/developing-blocks), or
-- interested in [embedding blocks in your application](https://blockprotocol.org/docs/embedding-blocks)
+The following pages provide a practical introduction for developers who want to:
+
+- [develop blocks](https://blockprotocol.org/docs/developing-blocks) and publish them to the block registry
+- [embed blocks](https://blockprotocol.org/docs/embedding-blocks) in their own application
 
 For the full technical details, read [the Block Protocol specification](https://blockprotocol.org/spec).
 
+### What's a block?
+
+Blocks are the basic units we use to build pages on the modern web. They are rectangular, full-width sections stacked on top of one another – also known as "block-level" elements.
+
+Blocks can be created using plain HTML, or by using a block editor interface like Notion or Wordpress' Gutenberg. With block editors you can usually rearrange the order of blocks on a page using drag and drop.
```

Comment:
```suggestion
Blocks can be inserted using plain HTML/JavaScript, or through visual interfaces provided by modern content management systems and applications like all-in-one workspaces. With these block-based editors you can usually rearrange the order of blocks on a page using drag and drop.
```

Reply:
Updated this to read:

> Blocks can be added to webpages using plain HTML and/or JavaScript. Some modern content management systems (eg. Wordpress, Squarespace) and applications (eg. Notion, Coda) allow you to add blocks to a page using a visual interface. With these block-based editors you can usually rearrange the order of blocks on a page using drag and drop. Blocks can be stacked on top of one another, arranged in columns, or embedded inside other blocks.

I know pointing to specific apps isn't ideal, so could remove the examples here. It would help paint a more concrete picture in people's minds of what we're talking about though.